mmv1/products/kms/CryptoKeyVersion.yaml

```diff
@@ -39,6 +39,7 @@ examples:
 custom_code: !ruby/object:Provider::Terraform::CustomCode
   custom_delete: templates/terraform/custom_delete/kms_crypto_key_version.erb
   custom_import: templates/terraform/custom_import/kms_crypto_key_version.go.erb
+  pre_update: templates/terraform/pre_update/kms_crypto_key_version.go.erb
```

Comment:
I _think_ `update_mask: true` will need to be removed as well, but I will run the CI anyway to see